cockroachcloud/backups-page.md

```diff
@@ -19,8 +19,8 @@ The backups that Cockroach Labs runs for you can be viewed on the [Backups page]
 Cockroach Labs runs [full cluster backups](../{{site.versions["stable"]}}/take-full-and-incremental-backups.html#full-backups) hourly for every {{ site.data.products.serverless }} cluster. The full backups are retained for 30 days.
 </section>
 
-<section class="filter-content" markdown="1" data-scope="{{ site.data.products.dedicated }}">
-Cockroach Labs runs [full backups](../{{site.versions["stable"]}}/take-full-and-incremental-backups.html#full-backups) daily and [incremental backups](../{{site.versions["stable"]}}/take-full-and-incremental-backups.html#incremental-backups) hourly for every {{ site.data.products.db }} cluster. The full backups are retained for 30 days, while incremental backups are retained for 7 days.
+<section class="filter-content" markdown="1" data-scope="dedicated">
```

Comment:
The scope here had the liquid variable in it... this was causing the following paragraph to not render. I can't take credit for this! @ashleykwtam noticed this in her PR #15524. Have added that change here since this PR is a quick fix to the Cloud/Dedicated variables that are incorrect. Thanks Ashley!

Reply:
Nice one! Good sleuthing Ashley! 🎉